### ISSUE TITLE
Animation driving component

### ISSUE THREAD

**Issue by jonnor** [microflo/microflo#71]
Wanted for c-base/ingress-table#28 and c-base/ingress-table#29

    inports:

    Value0| any Packet, or id+value Packets
    Value1| " - "
    Period| integer
    Duration| integer
    Dutycycle| integer (percentage)
    Start| integer
    Clock| integer

    outports:
    Value| Packet
    Running| boolean

* Driven by Clock, all other ports are non-triggering
* Sends Value0 at Start, and when Start+(Period*(dutycyle/100)) sends Value1, and when Start+Period sends Value0 again unless > Start+Duration.
* If Duration is < 0, continues forever.
* Sends true on Running when within duration, else false

Basically the thing is a square-wave function which takes Clock as input

**Comment by jonnor** [comment]
The component could perhaps support bracketed IPs, such that there is a set of N "tracks", all taking values shown above, driven by the same clock. Avoids having to have one component per "track". Problem with this not having a fixed/static memory use per component.

Pain of having one component per track might be significantly lowered if there are good ways of using an integer identifier for each track. Could possibly be handled by a subgraph with a Router in front, and a "Grouper" afterwards. Grouper would hold one ID Packet per inport, and when data comes in on inport, emit [bracket, id, data, endbracket]

**Comment by jonnor** [comment]
For use in the table we probably need a multi-track component though.
Could/would wrap this component and use bracket based indices for individual tracks...
